src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+// Footer.js
+
+const Footer = () => {
+    return (
+        <footer>
+            <p>🐾Juno College © 2022🐾</p>
+            
```

Comment:
Maybe have an a tag to the Juno Website and an another to your portfolio? with target ="blank" so it opens in a new tab. Just think you did way too good of a job to not have given yourself a credit tag